testcase/example.spec.js

```diff
@@ -0,0 +1,131 @@
+let homepage = require('../pageobjects/homepage');
+let newYoga = require('../pageobjects/newyoga');
+let items = require('../pageobjects/items');
+let helenaHoodedFleece = require('../pageobjects/helenahoodedfleece');
+let sale = require('../pageobjects/sale');
+let teesOnSale = require('../pageobjects/teesonsale');
+let karissaVNeckTee = require('../pageobjects/karissavnecktee');
+let elisaEvercoolTee = require('../pageobjects/elisaevercooltee');
+let comparingProducts = require('../pageobjects/comparingproducts');
+let bags = require('../pageobjects/bags');
+let overnightDuffleBag = require('../pageobjects/overnightdufflebag');
+let search = require ('../pageobjects/search');
+let checkout = require ('../pageobjects/checkout');
+let placeOrder= require('../pageobjects/placeorder');
+let createAccount1 = require('../pageobjects/createaccount1');
+let createAccount2 = require('../pageobjects/createaccount2');
+
+
+describe('demoapp', function() {
+
+  beforeEach(function() {
+
+      browser.waitForAngularEnabled(false);
+      homepage.get('https://magento.softwaretestingboard.com/');
+      browser.manage().window().maximize();
+
+    });
+
+  beforeAll(() => {
+      originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+      jasmine.DEFAULT_TIMEOUT_INTERVAL = 200000; 
+    });
+
+  afterAll( async () => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = originalTimeout;
+    });
+
+  getRandomString = function(length) {
+      var string = '';
+      var letters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz' 
```

Comment:
what do we use this for?

Reply:
I used it when creating new account because email should always be different, so with this function I randomize part of email every time test case is executed.